css/style.css

```diff
@@ -9,400 +14,622 @@ body {
-        width: 50%;
-        margin-left: 3%;
+@media (max-width: 961px) { 
+   .new-collection__background {
+    max-width: 45%;
+    min-height: 900px;
     }
 }
 
-@media (max-width: 1100px) { 
-     .new-сollection-arrow {
-        width: 10%;
-    } 
-     }
+@media (max-width: 961px) { 
+  .logo {
+    margin-right: 20%;
+    }
+}
 
+@media (max-width: 961px) {
+  .new-сollection-main-link {
+    margin-left: 5%;
+   } 
+}
 
 @media (min-width: 800px) { 
-    .logo-mobil,
-    .galka,
-    .new-collection__background_girl {
-        display: none;
-        } 
+  .logo-mobil,
+  .galka,
+  .new-collection__background_girl {
+    display: none;
     } 
-
+} 
+ 
 @media (max-width: 799px) { 
-    .logo,
-    .lang,
-    .check,
-    .new-сollection__social,
-    .text,
-    .new-сollection__main-card,
-    .new-collection__background {
-         display: none;
-        } 
+  .logo,
+  .lang,
+  .check,
+  .new-сollection__social,
+  .text,
+  .new-сollection__main-card,
+  .new-collection__background {
+    display: none;
     } 
-
+} 
+ 
 @media (max-width: 800px) { 
-    .logo-mobil {
-          position: absolute;
-          width: 98px;
-          height: 37px;
-          padding: 5% 0 0 7%; 
-        }
-    }  
-
-@media (max-width: 799px) { 
-    .girl {
-        background: none
-     }
+  .logo-mobil {
+    position: absolute;
+    width: 98px;
+    height: 37px;
+    padding: 5% 0 0 7%;
     }
-
+}  
+ 
 @media (max-width: 799px) { 
-    .new-сollection {
-        background: url(../img/girl2.jpg) no-repeat;
-        background-size: cover;
-        background-position: 0 92px; 
-        width: 80%;
-        margin-left: 10%;
-        position: absolute;
-        
-    }
- }
-
- @media (max-width: 799px) { 
-    .title {
-        top: 80%;
-        width: 15%;
-        font-size: 50px;
-        line-height: 52px;
-        left: 30%;
+  .girl {
+    background: none;
     }
-     } 
-
+}
+ 
+@media (max-width: 799px) { 
+  .new-сollection {
+    background: url(../img/girl2.jpg) no-repeat;
+    background-size: 125%;
+    background-position: 0 92px;
+     }
+}
+ 
+@media (max-width: 799px) {
+  .title {
+    top: 80%;
+    width: 15%;
+    font-size: 50px;
+    line-height: 52px;
+    left: 30%;
+     }
+} 
+ 
 @media (max-width: 800px) {      
-     .galka  {
-        position: absolute;
-        left: 45%;
-        top: 95%;
-    }
+  .galka  {
+    position: absolute;
+    left: 45%;
+    top: 95%;
+     }
 } 
-
+ 
 @media (max-width: 630px) { 
-    .new-сollection {
-        background: url(../img/girl2.jpg) no-repeat;
-        background-size: cover;
-        background-position: -50px 92px; 
-    }
- }
-
- @media (max-width: 630px) { 
-    .title {
-        top: 83%;
-        width: 15%;
-        font-size: 40px;
-        line-height: 42px;
-        left: 20%;
-    }
-     } 
-
+  .new-сollection {
+    background: url(../img/girl2.jpg) no-repeat;
+    background-size: cover;
+    background-position: -50px 92px; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)